narwhals/typing.py

```diff
@@ -47,11 +51,17 @@ def __dataframe__(self, *args: Any, **kwargs: Any) -> Any: ...
 Frame: TypeAlias = Union["DataFrame[Any]", "LazyFrame[Any]"]
 """Narwhals DataFrame or Narwhals LazyFrame"""
 
+IntoSeries: TypeAlias = Union[
+    "Series", "DataFrame[Any]", "LazyFrame[Any]", "NativeSeries"
+]
```

Comment:
sorry, the question didn't go for some reason

I wanted to ask, why include `"DataFrame[Any]", "LazyFrame[Any]"` here?

Reply:
I don't recall any particular reason; I just defined it to pass all checks and seem to have forgotten to restrict it again.

Let's try how about remove them.

Reply:
Seems fine!